src/app/dashboard/[id]/Column/Column.tsx

```
@@ -1,18 +1,48 @@
 import Image from 'next/image'
+import { useState } from 'react'
 
 import useCards from '@/app/api/useCards'
 import type { Column } from '@/app/api/useColumns'
+import { cn } from '@/app/shared/lib/cn'
 
+import { useCardMutation } from '../api/useCardMutation'
 import Card from '../Card/Card'
+import { useDragStore } from '../store/useDragStore'
 export default function Column({ column }: { column: Column }) {
```

Biome (1.9.4)

[error] Shouldn't redeclare 'Column'. Consider to delete it or rename it.

'Column' is defined here:


(lint/suspicious/noRedeclare)